### ISSUE TITLE
SCM icon badges when using change sets

### ISSUE THREAD

**Issue by julesx** [JohnstonCode/svn-scm#280]
**Is your feature request related to a problem? Please describe.**
The vscode SCM icon badge displays the number of changed files.  However, as soon as you add some changed files to a changelist, they are no longer represented in the badge.

**Describe the solution you'd like**
Adding files to changelists should not alter the count of changed files represented by the SCM icon badge.

**Describe alternatives you've considered**
Alternative would be showing # of changesets, but not sure how that would work when also having changed files not part of any changesets.

**Additional context**
none

**Comment by JohnstonCode** [comment]
Yeah, maybe this should be a config option as well. But I agree with your point moving things to change lists should still be counted on the badge.
